src/js/components/awardv2/visualizations/amounts/AggregatedAwardAmounts.jsx

```diff
@@ -49,10 +50,20 @@ export default class AggregatedAwardAmounts extends React.Component {
             const awardAmounts = this.props.awardAmounts;
             const visualizationType = determineScenario(awardAmounts);
             let visualization;
+            let extraRow = null;
             switch (visualizationType) {
                 case ('normal'):
                     visualization = (<NormalChart awardAmounts={awardAmounts} />);
                     break;
+                case ('exceedsCurrent'):
+                    visualization = (<ExceedsCurrentChart awardAmounts={awardAmounts} />);
+                    extraRow = (
+                        <div className="award-amounts__data-content">
+                            <div><span className="award-amounts__data-icon award-amounts__data-icon_overspending" />Exceeds Combined Current Award Amounts</div>
+                            <span>{awardAmounts.overspending}</span>
+                        </div>
+                    );
+                    break;
                 default:
```

Comment:
I know there are performance benefits to keeping stuff in the render but I wonder for readability if there's a point where it might be good to put some of the logic outside the render in a class method?

Reply:
Moved to the `generateVisualization` method.

Reply:
👌